Readme.md

```diff
@@ -210,6 +210,14 @@ Options are:
     -i, --isolate                    Do not run any other tests in the group used by --single(-s).
                                      Automatically turned on if --isolate-n is set above 0.
         --isolate-n                  Number of processes for isolated groups. Default to 1 when --isolate is on.
+        --specify-groups [SPECS]     Use 'specify-groups' if you want to specify multiple specs running in multiple
```

Comment:
won't this get super long / ugly ?
so it should read from a file ?

Reply:
What do you think about making there be 2 params, `--specify-groups` with the `,|` syntax, and `--specify-groups-from-file` which reads from file, comma delimited specs on the same line will run in 1 process (in that order), line breaks signifies new process

Reply:
that can be it's own PR if someone needs it :)